getit/getit.py

```diff
@@ -1,6 +1,8 @@
 import configparser
 import subprocess
 import sys
+import platform
+import os
```

Comment:
This package is also not required.. If we are not using it..

Reply:
and please fix flake8 errors.

Reply:
flake8 shows no errors

Reply:
Ok